release_notes.md

```diff
@@ -15,4 +15,8 @@ This file tracks release notes for the loop client.
 
 #### Breaking Changes
 
+* Macaroon authentication has been enabled for the `loopd` gRPC and REST
```

Comment:
Do you want to add why this (in combination with tls) is good?

Reply:
I mean something about remote node support

Reply:
Added a comment about exposing the port to the internet.